src/components/cylc/Info.vue

```diff
@@ -0,0 +1,261 @@
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+  <div class="c-info">
+    <!-- The task summary
+
+    * The "-40" comes from the GraphNode offset, see the GraphNode compoent.
+    * The height is "200" + "40".
+    * The "99999" is the maximum component width (because we don't know
+      how wide it will be until we render it).
+    * The "overflow-x: hidden" prevents this 99999 width from causing
+      horizontal scrolling.
+    * The "8em" is the height that this SVG will be scaled to fill.
+    -->
+    <div style="overflow-x: hidden;">
+      <svg
+        preserveAspectRatio="xMinYMin"
+        viewBox="-40 -40 99999 200"
+        height="6em"
+      >
+        <GraphNode
+          :task="task"
+          :jobs="task.children"
+          :jobTheme="jobTheme"
+        />
+      </svg>
+    </div>
+
+    <!-- The task information -->
+    <v-expansion-panels
+      multiple
+      variant="accordion"
+      v-model="panelExpansionModel"
+    >
+      <!-- The metadata -->
+      <v-expansion-panel class="metadata-panel">
+        <v-expansion-panel-title color="blue-grey-lighten-1">
+          Metadata
+        </v-expansion-panel-title>
+        <v-expansion-panel-text>
+          <dl>
+            <dt>Title</dt>
+            <dd>{{ taskMetadata.title }}</dd>
+            <v-divider />
+            <dt>Description</dt>
+            <dd><span class="markup">{{ taskMetadata.description }}</span></dd>
+            <v-divider />
+            <dt>URL</dt>
+            <dd>
+              <!--
+                  NOTE: for security reasons, always display the full URL
+                  that the link directs to
+              -->
+              <a
+                v-if="taskMetadata.URL"
+                :href="taskMetadata.URL"
+                target="_blank"
+              >
+                {{ taskMetadata.URL }}
+              </a>
+            </dd>
+            <v-divider />
+            <template v-for="(value, key) in customMetadata" :key="key">
+              <dt>{{ key }}</dt>
+              <dd><span class="markup">{{ value }}</span></dd>
+              <v-divider />
+            </template>
+          </dl>
+        </v-expansion-panel-text>
+      </v-expansion-panel>
+
+      <!-- The prereqs -->
+      <v-expansion-panel class="prerequisites-panel">
+        <v-expansion-panel-title color="blue-grey-lighten-2">
+          Prerequisites
+        </v-expansion-panel-title>
+        <v-expansion-panel-text>
+          <ul>
+            <li v-for="prereq in prerequisites" :key="prereq.expression">
+              <span
+                class="prerequisite-alias condition"
+                :class="{satisfied: prereq.satisfied}"
+              >{{ prereq.expression }}</span>
```

Comment:
The prereq expression is a little confusing - why is `c` in it? 

![image](https://github.com/user-attachments/assets/e88cedc1-f3e2-49d1-a9ec-c1486b7cc8b8)

Reply:
This is (currently) how Cylc represents conditional expressions. Please try using the `cylc show` command which this view is replicating.

At the moment, this is the only way the GUI can represent a conditional expression. I have opened an issue to look at improving this, see #1893 for more deets.

Reply:
`cylc show` does not have `c` in it, just numbers

Reply:
Interesting. The `c` prefix is present in the GraphQL response:

```graphql
query {
  workflows {
    taskProxies {
      id
      prerequisites {
        satisfied
        expression
        conditions {
          taskId
          message
          reqState
          exprAlias
          satisfied
        }
      }
    }
  }
}
```

E.G:

```
{
  "data": {
    "workflows": [
      {
        "taskProxies": [
          {
            "id": "~me/show/run1//1/b",
            "prerequisites": []
          },
          {
            "id": "~me/show/run1//1/x",
            "prerequisites": [
              {
                "satisfied": false,
                "expression": "c0 & (c1 | c2) & (c3 | c4)",
                "conditions": [
                  {
                    "taskId": "1/a",
                    "message": "satisfied naturally",
                    "reqState": "succeeded",
                    "exprAlias": "c0",
                    "satisfied": true
                  },
```

I guess we could strip the first character? Presumably `cylc show` does that itself?

Reply:
Yep, `cylc show` strips the `c`:

https://github.com/cylc/cylc-flow/blob/b8be2e3ef5b37288df5de4d62642b0d84e8eb02d/cylc/flow/scripts/show.py#L315

I'll update this to do the same.

Reply:
Done.

Reply:
Here is where the `c` comes from (suddenly remembered seeing this on my `cylc-remove` work): https://github.com/cylc/cylc-flow/blob/63a6a377982eb91875a260e6b734435b3ae00740/cylc/flow/prerequisite.py#L239

Don't know why we can't just remove it?

Reply:
`cylc show` uses `string.strip('c')` and I've used `string.replace(/c/g, '')` here, so we should be safe to remove it.

Reply:
For #1893, I think we should remove the "alias" as a user-facing thing and find another way to display it (we're not bound by plain text here).